www/js/diary/detail.js

```diff
@@ -42,11 +43,17 @@ angular.module('emission.main.diary.detail',['ui-leaflet', 'ng-walkthrough',
       console.log("diary/detail received resize event, invalidating map size");
       data.leafletObject.invalidateSize();
   });
-
+  $scope.name = "Cannot Retrieve Suggestion";
+  $scope.mode = "Cannot Retrieve Mode";
+  $scope.bid = "yalis-stanley-hall-cafe-berkeley";
```

Comment:
I am glad you were able to test with a hardcoded value. But this code needs to be removed after testing! Otherwise, you will end up with a situation where your users see the yali's reviews even when they don't have a suggestion for yali's!